core/breakpoints/manager.php

```diff
@@ -534,4 +534,36 @@ private static function get_stylesheet_templates() {
 
 		return apply_filters( $replacement_hook, $templates );
 	}
+
+	/**
+	 * Get file handle ID.
+	 *
+	 * Retrieve the handle ID for the post CSS file.
+	 *
+	 * @since 3.15.0
+	 * @access protected
+	 *
+	 * @return string CSS file handle ID.
+	 */
+	protected function get_file_handle_id() {
+		$current_document = Plugin::$instance->documents->get_current();
+		if ( $current_document ) {
+			$post_id = $current_document->get_main_id();
+			return 'elementor-post-' . $post_id;
+		}
+
+		return false;
+	}
+
+	public function __construct() {
+		add_action( 'elementor/css_file/parse_content', function() {
+			$handle_id = $this->get_file_handle_id();
```

Comment:
```suggestion
		add_action( 'elementor/css_file/parse_content', function( $css_file ) {
			$handle_id = $css_file->get_file_handle_id();
```

Reply:
As a result of these changes, it looks like this :
Template widget and shortcode widget stop generating responsive controls, like in this initial state. 

I'm looking into this at the moment